src/site/_includes/partials/analytics.njk

```diff
@@ -10,12 +10,16 @@ ga('set', 'transport', 'beacon');
 ga('set', 'anonymizeIp', true);
 ga('set', 'page', window.location.pathname);
 ga('set', '{{ site.analytics.dimensions.TRACKING_VERSION }}', '{{ site.analytics.version}}');
-try {
-  ga('set', '{{ site.analytics.dimensions.NAVIGATION_TYPE }}', performance.getEntriesByType('navigation')[0].type.replace(/_/g, '-'));
-} catch (error) {
-  ga('set', '{{ site.analytics.dimensions.NAVIGATION_TYPE }}', '(not set)');
-}
 ga('set', '{{ site.analytics.dimensions.COLOR_SCHEME_PREFERENCE }}', window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light');
-ga('send', 'pageview');
+// Do not send a pageview on pages which are prerendering
+// These will be handled later once the page becomes active
+if (!document.prerendering) {
```

Comment:
As I mentioned in the other comment, it's possible that this code will run after the page has already been activated, which means this conditional will return true and a normal pageview will be sent without the `prerender` type set.

I don't think there's any good way to do this without either:

* Doing all the prerender checking in the template code, or
* Moving all of the pageview sending logic from the template into the analytics file (which is ultimately a better place for it)